distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -16,6 +16,11 @@ It defines:
 
 See <<{reference-prefix}org.codice.ddf.security.policy.context.impl.PolicyManager, Web Context Policy Manager Configurations>> for detailed descriptions of all fields.
 
+=== Guest Access
+Guest access is a toggleable configuration.
+Allowing guest access will allow the use of the guest principal throughout the entire system.
```

Comment:
> ... allow the use of the guest principal throughout the entire system

✏️ I feel like this is a bit vague. Maybe "Enabling guest access will cause all users to be assigned a guest principle..."

Reply:
✏️ Whoops, `principle` should be `principal`